### DISCUSSION TITLE
How can I show Output JSON data on Editorjs again?

### DISCUSSION THREAD

**Discussion by theaungmyatmoe** [codex-team/editor.js#1889] [Q&A]
I am trying to show Editorjs Output json on Editorjs again.
I tried with `render` method , it is not working.

I want to show my JSON on Editorjs in edit mode.

Like this.

```html
  <div id="editorjs">
    </div>

  <script src="{{ asset('js/app.js')}}"></script>
  <script>
    const editor = new EditorJS({
      editor.render(
blocks: [
        {
          type: "image",
          data: {
            url: "https://cdn.pixabay.com/photo/2017/09/01/21/53/blue-2705642_1280.jpg"
          }
        },
        {
          type: "header",
          data: {
             text: "New header",
             level: 2
          }
        }
      ]
    });
  }
     )
    </script>
```

And it should show like this.
![Screenshot_2021-03-12-22-44-15-18](https://user-images.githubusercontent.com/65492233/110967189-824df180-8384-11eb-8a9f-58efa2554fbe.jpg)

How should I render it?

**Comment by entonee** [comment]
use - data instead of render.

data: {
   "time": 1550476186479,
   "blocks": [
      {
         "type": "header",
         "data": {
            "text": "Editor.js",
            "level": 2
         }
      }
]},

**Comment by theaungmyatmoe** [comment]
> use - data instead of render.
> 
> data: {
> "time": 1550476186479,
> "blocks": [
> {
> "type": "header",
> "data": {
> "text": "Editor.js",
> "level": 2
> }
> }
> ]},

I will try your idea very soon,thank bro.

**Comment by smostafad** [comment]
Hello , I want to change the json file dynamically, that is, the user fills in the information on the site and downloads the new json file. Can you help me?
